### PR TITLE
Develop stream 20200121

### PR DESCRIPTION
**Pull request by neon60**
- Improve readme
- Add half-overload for load/store-volatile
- Gitlab CI fix
- Fix compile warnings
- Added settable seed option

### CODE REVIEW

**Comment by saadrahim** [comment]
Is there an documentation on the settable seeds?

File: test/rocprim/test_seed.hpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef TEST_SEED_HPP_
+#define TEST_SEED_HPP_
+
+static constexpr bool use_seed = true;
```
Comment (saadrahim) [inline test/rocprim/test_seed.hpp]: Add random seeds number. Random seeds and known should be part of testing

Comment (neon60) [reply test/rocprim/test_seed.hpp]: Resolved.

**Comment by neon60** [comment]
> Is there any documentation on the settable seeds?

Added a section to readme.

**Pull request merged**